datasets/hk/legco/hk_legco.yml

```diff
@@ -0,0 +1,38 @@
+name: hk_legco
+title: Members of the Hong Kong Legislative Council
+url: https://www.legco.gov.hk/en/members/legco-members/members-biographies.html
+summary: >-
+  Members of the Hong Kong Legislative Council.
+description: |
+  Members of the Hong Kong Legislative Council.
+
+  This dataset lists the current members of the Legislative Council of
+  the Hong Kong Special Administrative Region of the People's
+  Republic of China by their English and Chinese names.
+entry_point: crawler.py
+prefix: hk-legco
+publisher:
+  name: >-
+    Hong Kong Legislative Council
+  official: true
+  description: |
+    Legislative Council of the Hong Kong Special Administrative Region
+    of the People's Republic of China
+
+    This dataset is collected from the API for the official website of
+    the Legislative Council, which is the legislative body for the
+    Hong Kong SAR.  The Legislative Council is composed of 90 members,
+    20 of whom are popularly elected by geographical constituencies,
+    30 of whom are elected by specific functional constituencies
+    composed of organizations or individuals in specific segments of
+    Hong Kong society, and 40 of whom are elected by the 1500-member
+    Election Committee.
+
+    It includes the current members along with their names (in
+    English, Traditional and Simplified Chinese), contact information,
+    and educational qualifications.
```

Comment:
All of this can move to the dataset description.

Very nice in general though!

Reply:
Ah, of course - I went through the metadata documentation again and I think it's correct now.